src/main/resources/assets/commandblockide/lang/de_de.json

```
@@ -0,0 +1,23 @@
+{
+    "arguments.editfunction.tag.unsupported": "Kann keine Funktions Tags bearbeiten",
+    "commands.editfunction.failed.modNotInstalled": "Die Command Block IDE muss installiert sein",
```

```suggestion
    "commands.editfunction.failed.modNotInstalled": "Die Command Block IDE Mod muss installiert sein",
```
In the English version there is a `mod` after the `Command Block IDE`.